ResearchKit/Consent/ORKVisualConsentStepViewController.m

```diff
@@ -661,13 +662,12 @@ - (NSUInteger)indexOfViewController:(UIViewController *)viewController {
         return NSNotFound;
     }
     
-    NSUInteger index = NSNotFound;
     for (NSNumber *key in _viewControllers) {
         if (_viewControllers[key] == viewController) {
-            index = [key unsignedIntegerValue];
+            return [key unsignedIntegerValue];
```

Comment:
I personally prefer having a `break` here, instead of two `return`s (a single method exit point is easier to read). Like you did in `-scrollView`. ;-)